devops/scripts/aad/get_existing_app.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
-set -euo pipefail
```

Comment:
why we don't need this one ? We still want to have an error if any of the commands failed, don't we?

Reply:
As discussed above, this script file contains shared function definitions and is intended to be `source`d into the parent context.

When invoking a script normally, these directives only apply to the context of the script they are in. For a `source`d script, they apply to the caller's context and we should leave it to the caller to determine the behaviour they want (the calling scripts _do_ all have this set 😄 )